### ISSUE TITLE
jetbrains plugin:Qwen 2.5:  suffix is not currently supported

### ISSUE THREAD

**Issue by mkhludnev** [carlrobertoh/CodeGPT#720]
### What happened?

I have Qwen 2.5 deployed internally. The recent plugin in set into JetBrains Idea. Everything is almost fine - thanks for the great project. However, default CodeCompletion settings is little bit odd 
![Screenshot from 2024-10-02 14-29-22](https://github.com/user-attachments/assets/f9e4c1a7-18c2-48ab-8c09-cb8caecd7de3)
it causes 

```
Unknown API response. Code: 400, Body: {"object":"error","message":"suffix is not currently supported","type":"BadRequestError","param":null,"code":400}

java.lang.RuntimeException
  at ee.carlrobert.llm.completion.CompletionEventSourceListener.onFailure(CompletionEventSourceListener.java:116)
```
after I remove suffix property and set the prompt, code completions seem working now
![Screenshot from 2024-10-02 14-31-39](https://github.com/user-attachments/assets/0dca8dba-7870-4769-8f46-00a1a9ef56de)
 

### CodeGPT version

2.11.4-241.1

### Operating System

Linux

**Comment by linpan** [comment]
Qwen2.5-Coder-7B-lnstruct suffix is not currently supported 
![CleanShot 2024-10-05 at 13 49 30](https://github.com/user-attachments/assets/b1274ac2-3f39-443f-abfa-719a2142291f)

**Comment by carlrobertoh** [comment]
Yes, I can confirm that this is a correct behaviour. The request body must contain either a raw prompt via the `FIM_PREFIX` placeholder or `PREFIX` and `SUFFIX` together. The suffix placeholder was added primarily for OpenAI text completions purposes.

Thanks for bringing it up, this needs to be documented better.
